runtime/rasdump/dmpagent.c

```diff
@@ -655,6 +671,21 @@ doSilentDump(J9RASdumpAgent *agent, char *label, J9RASdumpContext *context)
 	return OMR_ERROR_NONE;
 }
 
+static omr_error_t
+doJavaVMExit(J9RASdumpAgent *agent, char *label, J9RASdumpContext *context)
+{
+	J9JavaVM *vm = context->javaVM;
+	J9VMThread *vmThread = vm->internalVMFunctions->currentVMThread(vm);
+
+	PORT_ACCESS_FROM_JAVAVM(vm);
+	j9nls_printf(PORTLIB, J9NLS_INFO | J9NLS_STDERR, J9NLS_DMP_EXIT_SHUTDOWN, 
+		context->eventData != NULL ? context->eventData->detailLength : 0, 
+		context->eventData != NULL ? context->eventData->detailData : NULL);
```

Comment:
I'm thinking this should print something like "unknown" (from NLS data) rather than printing nothing. The output string is `Reason: %2$.*1$s` so if there is no eventData we'll get `Reason: ` or `Reason: NULL`.